tasks/combinedfields.tasks

```diff
@@ -0,0 +1,1500 @@
+CFQHighHigh: several following +combinedFields=titleTokenized^4.0,body^2.0 # freq=436129 freq=416515
```

Comment:
Let's make this tasks file smaller by only taking queries from the `wikinightly.tasks` tasks file?

Reply:
Done. I've adapted the `OrHighHigh` and `OrHighMed` queries from `wikinightly.tasks`, as well as the last 5 `OrHighLow` queries from another task file since the nightly one doesn't contain low freq term queries.